VPN support update (#2509)
saucelabs/sauce-docs: docs/mobile-apps/real-device-cleaning.md

Every 5th cleaning session includes a device reboot. This ensures that the system remains
performant and any stray processes are stopped.

:::caution

> On Public Devices setting up VPNs is not supported. Please check our documentation related to settings up a secure connection to your internal networks/firewall/endpoints. [Sauce Connect ](secure-connections/sauce-connect-5/) for details.

Reviewer: ```suggestion
Setting up VPNs on public devices is not supported. Please refer to our documentation for guidance on establishing a secure connection to your internal networks, firewalls, or endpoints. See [Sauce Connect](secure-connections/sauce-connect-5/) for more details.
```

> Setting up VPNs on public devices is not supported. Please refer to our documentation for guidance on establishing a secure connection to your internal networks, firewalls, or endpoints. See [Sauce Connect](secure-connections/sauce-connect-5/) for more details.

Reviewer: ```suggestion
Setting up VPNs on public devices is not supported. Please refer to our documentation for guidance on establishing a secure connection to your internal networks, firewalls, or endpoints. See [Sauce Connect](./secure-connections/sauce-connect-5/) for more details.
```

On Private Devices, access to some parts of the device settings is restricted: Wifi and Network Settings, Profile and Device Management, Passcode/Biometrics Settings, Developer options, Resetting device options, Creating AppleIDs, Battery Options, and Display Settings.  These rules apply to both iOS and Android public devices.
:::

:::caution

> On Private Devices setting up VPNs is not supported. Please check our documentation related to settings up secure connection to your internal networks/firewall/endpoints. [Sauce Connect ](secure-connections/sauce-connect-5/) for details.

Reviewer: ```suggestion
Setting up VPNs on private devices is not supported. Please consult our documentation on how to set up a secure connection to your internal networks, firewalls, or endpoints. For more details, visit [Sauce Connect](secure-connections/sauce-connect-5/).
```

> Setting up VPNs on private devices is not supported. Please consult our documentation on how to set up a secure connection to your internal networks, firewalls, or endpoints. For more details, visit [Sauce Connect](secure-connections/sauce-connect-5/).

Reviewer: ```suggestion
Setting up VPNs on private devices is not supported. Please consult our documentation on how to set up a secure connection to your internal networks, firewalls, or endpoints. For more details, visit [Sauce Connect](./secure-connections/sauce-connect-5/).
```
